ci/run_envoy_docker.sh

```diff
@@ -11,15 +11,13 @@ else
 	USER_GROUP=$(id -g)
 fi
 
-# The IMAGE_ID can be set to an arbitrary image ID (found with 'docker images').
-[[ -z "${IMAGE_ID}" ]] && IMAGE_ID="latest"
+# The IMAGE_ID defaults to the CI hash but can be set to an arbitrary image ID (found with 'docker
+# images').
+CI_ENVOY_BUILD_SHA=$(grep "^ENVOY_BUILD_SHA" ci/ci_steps.sh | cut -d\= -f 2)
```

Comment:
nit: instead of this fragile cut, can we potentially just move ENVOY_BUILD_SHA into envoy_build_sha.sh and source it from both places? (Then also easier to find where to update the SHA)